examples/chip-tool-darwin/templates/README.zapt

```diff
@@ -0,0 +1,145 @@
+
+## Using the Client to Send Matter Commands
+
+To use the Client to send Matter commands, run the built executable and pass it
+the target cluster name, the target command name as well as an endpoint id.
+
+The endpoint id must be between 1 and 240.
+
+    $ chip-tool-darwin onoff on 1
+
+The client will send a single command packet and then exit.
+
+### How to get the list of supported clusters
+
+To get the list of supported clusters, run the built executable without any
+arguments.
+
+    $ chip-tool-darwin
+
+Example output:
+
+```bash
+Usage:
+  ./chip-tool-darwin cluster_name command_name [param1 param2 ...]
+
+  +-------------------------------------------------------------------------------------+
+  | Clusters:                                                                           |
+  +-------------------------------------------------------------------------------------+
+  | * barriercontrol                                                                    |
+  | * basic                                                                             |
+  | * colorcontrol                                                                      |
+  | * doorlock                                                                          |
+  | * groups                                                                            |
+  | * iaszone                                                                           |
+  | * identify                                                                          |
+  | * levelcontrol                                                                      |
+  | * onoff                                                                             |
+  | * pairing                                                                           |
+  | * payload                                                                           |
+  | * scenes                                                                            |
+  | * temperaturemeasurement                                                            |
+  +-------------------------------------------------------------------------------------+
+```
+
+### How to get the list of supported commands for a specific cluster
+
+To get the list of commands for a specific cluster, run the built executable
+with the target cluster name.
+
+    $ chip-tool-darwin onoff
+
+### How to get the list of supported attributes for a specific cluster
+
+To the the list of attributes for a specific cluster, run the built executable
+with the target cluster name and the `read` command name.
+
+    $ chip-tool-darwin onoff read
+
+### How to get the list of parameters for a command
+
+To get the list of parameters for a specific command, run the built executable
+with the target cluster name and the target command name
+
+    $ chip-tool-darwin onoff on
+
+# Command Reference
+
+## Command Details
+
+<details><summary>All Clusters</summary><p>
+
+{{#chip_client_clusters}}
```

Comment:
Could we avoid having a ZAPT for readme files even?

This is too much generated code.

Reply:
I took Vivien's advice and placed the generated files to avoid any code gen conflicts.

Reply:
Please do not generate README.md 
The readme.md can describe how to use the tool and the tool can output list commands or autocomplete.

An autogenerated readme.md cannot really give user-friendly commands, it only is a flat list of available commands, so I am claiming the benefit we get from this is minimal as it is just a list that is not a friendly to humans as there is no detail on what commands do or what they are used for.

Reply:
Removed.